filebeat/docs/modules/cisco.asciidoc

```diff
@@ -278,6 +289,23 @@ include::../include/timezone-support.asciidoc[]
 
 :fileset_ex!:
 
+[float]
+[[dynamic-script-compilations]]
+=== Dynamic Script Compilations
+
+The `asa` and `ftd` filesets are based on ingest pipelines and make extensive
+use of script processors and painless conditions. This can cause the pipelines
+to fail loading the first time the module is used, due to exceeding the maximum
+script compilation limits. It is recommended to tune the following parameters
+on your cluster:
+
+- {ref}/circuit-breaker.html#script-compilation-circuit-breaker[script.max_compilations_rate]:
+  Increase the default value of `75/5m` to at least `100/5m`.
```

Comment:
Might be best to avoid mentioning the default here because defaults change over time, and this info is likely to get stale.

Reply:
Just to clarify... I mean that the ES docs should be the source of truth for things like defaults. We probably shouldn't mention them in the Beats docs if we can avoid it.

Reply:
makes complete sense, thanks